coast/TIDEGAUGE.py

```diff
@@ -383,31 +384,33 @@ def read_HLW_header(filnam):
         -------
         dictionary of attributes
         """
-        debug(f'Reading HLW header from "{filnam}" ')
+        debug(f"Reading HLW header from \"{filnam}\" ")
         fid = open(filnam)
```

Comment:
Another place to use path.read_text or a context manager.

Reply:
noted for future optimization requirement, if performance takes a hit.